cmake/Modules/FindCppCMS.cmake

```diff
@@ -0,0 +1,14 @@
+if (CPPCMS_INCLUDE_DIR AND CPPCMS_LIBRARY)
+
+  # in cache already
+  SET(CppCMS_FOUND TRUE)
```

Comment:
Actually I copied this from another project out there and added something, because it didn't work without. But I agree, it is (and works) weird.